### PR TITLE
Multisig beginner guide

### PR DESCRIPTION
**Pull request by relativisticelectron**
Issue: https://github.com/cryptoadvance/specter-desktop/issues/1728

The guide is (only) an overview of multisig. For details I refer to: https://bitcoiner.guide/multisig/

Changes in this PR:
- Added https://deploy-preview-1731--specter-desktop-docs.netlify.app/multisig-guide/
- Added video in Readme https://deploy-preview-1731--specter-desktop-docs.netlify.app/#video-walkthrough
- Added various links to the docs
- Fixed https://github.com/cryptoadvance/specter-desktop/issues/1736 that the mkdocs can render the faq properly even though only 2 spaces for sublists are used, and not 4
- Videos can now be placed with `small_video`  or `video` anywhere in the docs and https://pypi.org/project/mkdocs-video/ will embed it

- [x] Replace `https://deploy-preview-1731--specter-desktop-docs.netlify.app` by `https://specter-desktop-docs.netlify.app` after PR review

### CODE REVIEW

File: docs/multisig-guide.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+#### Seed storage locations
+
+This is a crucial part of the setup and needs **careful consideration**. Only if you choose these locations well you get the benefits (see [https://bitcoiner.guide/multisig/intro/#multisig-benefits](https://bitcoiner.guide/multisig/intro/#multisig-benefits)). Here a few points to pay attention to:
+
+* Seed backups (on steel) are not protected by a pin as the *signing device*s (e.g. *hardware wallets*). They therefore need a **secure location**, i.e. accessible only to you or people you completely trust.
+  * Examples are: Second home/apartment, Safe deposit box, a relatives (or very good friends) house/apartment
+* In 1 location there should not be more than 1 seed
+  * Minimum is: 3 **secure locations**, in each location is the signing device and its seed backup
+  * Best is: 3 **secure locations** for the seed backups **and** 3 (medium) secure locations for the *signing device*s (e.g. *hardware wallets*).
```
Comment (k9ert) [inline docs/multisig-guide.md]: I would add here, that ephemeral devices might reduce the number of backups you need. Also an emergency key might not even need a device. So e.g. one emergency key without a device and one ephemeral device reduces the number of secure locations to 4.

File: docs/multisig-guide.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Because the multisig setup is more complicated, testing all parts of the setup is even more important.  All the tests should be done only with **minimal funds** on the multisig wallet.
+
+#### 4a Test the seed backups
+
+Crucial is that you backed up each seed correctly (on steel). Examples:
+
+* Double and triple checking by comparing the steel backup with the *signing device*  (also known as *hardware wallet*)
+* a) Wiping the *signing device*  (also known as *hardware wallet*), b) restoring the  *signing device*  from the backup, c) exporting the  *(master) public key*  ensuring it is the same as before.
+
+#### 4b (Watch-only) Wallet test
```
Comment (k9ert) [inline docs/multisig-guide.md:103]: I think the pennytest shouldn't be mentioned last. Imho, it should even be combined with the seed-backup-test:
* After initial setup, send a small amount in
* wipe the setup
* recreate the setup from the backups and send the money out again

File: docs/multisig-guide.md (hunk 1 above)
Comment (relativisticelectron) [reply docs/multisig-guide.md]: Ok, I added a bullet point addressing this: https://github.com/cryptoadvance/specter-desktop/commit/c58a4bec0140f5278d62063fd94bc4e8072a6d83

File: docs/multisig-guide.md (hunk 2 above)
Comment (k9ert) [reply docs/multisig-guide.md]: It's not yet changed?!

Comment (relativisticelectron) [reply docs/multisig-guide.md]: What do you mean? I changed section 4.1 through 4.4.

Comment (k9ert) [reply docs/multisig-guide.md]: Did you forget to push? the pennytest is still mentioned last and the seed-backuptest in 4a does not mention the pennytest, yet?

Comment (relativisticelectron) [reply docs/multisig-guide.md]: Just to be sure, I merged the latest master into the branch https://github.com/cryptoadvance/specter-desktop/pull/1731/commits/e4330f8227b3e16bb6e93056fe9ffedb290ddfee and the https://deploy-preview-1731--specter-desktop-docs.netlify.app/multisig-guide/ looks like
image

I am unsure which commit you are referring to.

Comment (k9ert) [reply docs/multisig-guide.md]: Nono, all good! I clicked on the "View changes" button on the top of this comment and i have no idea what's the purpose of that button if it doesn't show the recent changes.

**Comment by k9ert** [comment]
The latest test-failure should be solved with the latest merge.

**Comment by djpnewton** [comment]
It seems like the multisig trade-offs doc should be merged into this one?

**Comment by k9ert** [comment]
Ready to merge when you are!

**Comment by relativisticelectron** [comment]
OK. I replaced https://deploy-preview-1731--specter-desktop-docs.netlify.app by https://specter-desktop-docs.netlify.app .  
It can now be merged.

**Comment by djpnewton** [comment]
There was a link on the create multisig wallet page
 Did you amend that link?

**Comment by djpnewton** [comment]
> There was a link on the create multisig wallet page Did you amend that link?

Ah yes it looks like it has been updated

**Pull request merged**